src/java/main/org/apache/zookeeper/server/auth/SASLAuthenticationProvider.java

```diff
@@ -38,11 +38,6 @@ public String getScheme() {
     }
 
     public boolean matches(String id,String aclExpr) {
-        if (System.getProperty("zookeeper.superUser") != null) {
-            if (id.equals(System.getProperty("zookeeper.superUser")) || id.equals(aclExpr)) {
-              return true;
-            }
-        }
         if ((id.equals("super") || id.equals(aclExpr))) {
```

Comment:
Sure.  It looks like I am going to have to pull out the "super" user SASL Login too.

https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/auth/SaslServerCallbackHandler.java#L42
https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/auth/SaslServerCallbackHandler.java#L99-L101

Do you know where the source for the documentation at https://zookeeper.apache.org/doc/r3.3.3/zookeeperAdmin.html is? Under the section "Authentication & Authorization Options" it talks about how to use the "super" user. 

I also found docs for this in a few other places on the open internet and because this is a backwards incompatible change I get a little nervous, so is there anything else I need to do to document a breaking change like this?